tasks/main.yml

```diff
@@ -1,7 +1,9 @@
 ---
 
-- include_vars: "{{ ansible_distribution | replace(' ', '_') }}.yml"
-  ignore_errors: yes
+- name: Detect Seapath distribution
+  include_role:
```

Comment:
I prefer avoiding creating dependencies between roles.

If possible, it will be great to detect the udev service name automatically.

We do that with the service fact: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/service_facts_module.html